### DISCUSSION TITLE
Lilypond Parser from Scratch?

### DISCUSSION THREAD

**Discussion by jgarte** [Abjad/abjad#1503] [Ideas]
@trevorbaca 

Would it be too crazy of an idea to rewrite write the [Lilypond parser](https://github.com/Abjad/abjad/tree/main/abjad/parsers) without using [ply](https://github.com/Abjad/abjad/blob/c969b3c5476fe62f55e02eab9283d6c350b4f69f/setup.py#L76)?

**Comment by trevorbaca** [comment]
Yes.

**Comment by josiah-wolf-oberholtzer** [comment]
@jgarte What would be the benefit of rewriting?
